pkg/manager/member/executor_manager.go

```diff
@@ -778,29 +779,36 @@ func (m *executorMemberManager) syncExecutorMembersStatus(tc *v1alpha1.TiflowClu
 		return nil, err
 	}
 
-	return syncExecutorMembers(executorsInfo)
+	return syncExecutorMembers(tc, executorsInfo)
 }
 
-func syncExecutorMembers(executors tiflowapi.ExecutorsInfo) (map[string]v1alpha1.ExecutorMember, error) {
+func syncExecutorMembers(tc *v1alpha1.TiflowCluster, executors tiflowapi.ExecutorsInfo) (map[string]v1alpha1.ExecutorMember, error) {
 	// todo: WIP, get information about the FailureMembers and FailoverUID through the MasterClient
 	// sync executors info
+	ns := tc.GetNamespace()
 	members := make(map[string]v1alpha1.ExecutorMember)
 	for _, e := range executors.Executors {
+		// todo: WIP
+		if !strings.Contains(e.ID, ns) {
```

Comment:
🤔

Reply:
This is just the current filtering, which may need to be changed later, and cannot be avoided when `tcName` is equal to `ns`